app/models/concerns/outgoings_summarisable.rb

```diff
@@ -1,27 +1,31 @@
 module OutgoingsSummarisable
   def client_mortgage
-    net_housing_costs if client_income_relevant? && property_owned_with_mortgage?
+    property_owned_with_mortgage? ? net_housing_costs : 0
```

Comment:
These outgoings fields are now being filled out both if the client gets asylum support and if a client gets passporting benefit, which they shouldn't be.